src/core/tests/playwright/conftest.py

```diff
@@ -48,6 +48,8 @@ def browser_context_args(browser_context_args, browser_type_launch_args, request
 @pytest.fixture(scope="session")
 def taranis_frontend(e2e_server, browser_context_args, browser: Browser):
     context = browser.new_context(**browser_context_args)
+    # Drop timeout from 30s to 10s
```

Comment:
This is the default timeout for everything in playwright. So to say, if a test can't find an element, before it fails, it will wait 20s less which is to me better, 30s is an unreasonable time.